Userland/Services/WebContent/WebDriverConnection.cpp

```diff
@@ -1805,4 +1805,25 @@ void WebDriverConnection::delete_cookies(Optional<StringView> const& name)
     }
 }
 
+// https://w3c.github.io/webdriver/#dfn-get-computed-role
+Messages::WebDriverClient::GetComputedRoleResponse WebDriverConnection::get_computed_role(DeprecatedString const& element_id)
+{
+    // 1. If the current top-level browsing context is no longer open, return error with error code no such window.
+    TRY(ensure_open_top_level_browsing_context());
+
+    // 2. Handle any user prompts and return its value if it is an error.
+    TRY(handle_any_user_prompts());
+
+    // 3. Let element be the result of trying to get a known connected element with url variable element id.
+    auto* element = TRY(get_known_connected_element(element_id));
+
+    // 4. Let role be the result of computing the WAI-ARIA role of element.
+    auto role = element->role_or_default();
+
+    // 5. Return success with data role.
+    JsonObject result;
+    result.set("role"sv, role.view());
+    return result;
```

Comment:
This should just be:
```c++
return role;
```

If that doesn't compile (idk off hand if `DeprecatedFlyString` is convertible to a `JsonValue`), then:
```c++
return DeprecatedString { role };
```

In general "Return success with <something>" generally means, literally, "return <something>" from here.